src/reactor/mod.rs

```diff
@@ -171,8 +172,16 @@ impl Reactor {
             Err(e) => return Err(e),
         }
 
+        let start = if log_enabled!(Level::Debug) {
+            Some(Instant::now())
+        } else {
+            None
+        };
+
         // Process all the events that came in, dispatching appropriately
+        let mut events = 0;
         for i in 0..self.events.len() {
+            events = 1;
```

Comment:
This does not seem correct... I'm assuming that you are trying to get the event count?